portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -1,20 +1,95 @@
 ---
-page_title: Accounts | Portafly
-page_title_description: Page title of the Accounts page
-body_title: Accounts
-body_title_description": Body title of the Accounts page
-accounts_table:
-  aria_label: Accounts list
-  aria_label_description: Aria label of the Accounts table component
-  empty_state: There are no accounts
-  empty_state_description: Message of the Accounts table empty state
-  col_group: Group/org.
-  col_group_description: Title of the Group/org. column
-  col_admin: Admin
-  col_admin_description: Title of the Admin column
-  col_signup: Signup date
-  col_signup_description: Title of the Signup date column
-  col_apps: Apps
-  col_apps_description: Title of the Apps column
-  col_state: State
-  col_state_description: Title of the State column
+# Account settings > Users
+
+title_page: Accounts | Team members
+title_page_description: Title for the page to manage team members
+
+# Top area
+
+body_title: Team members
+body_title_description: Title for the main area to manage team members
+invite_new_member_button: Invite a new team member
+invite_new_member_button_description: Button link to send invitations to new team members
+
+# Main area
+
+## Tabs
+listing_tab: Listing
+listing_tab_description: Label for the tab that contains the list of all team members, selection criteria and actions that can be applied on them.
+invitations_tab: Invitations
+invitations_tab_description: Label for the tab that contains the list of all invitations, selection criteria and actions that can be applied on them.
+
+## Tab: Listing
+## Toolbar
+bulk_selector: {{number}} selected
+bulk_selector_description: Label for the bulk selector to indicate the number of chosen team members. This label is visible only when team members were selected.
+# Delete button: available in shared.yml
+team_members_filter: [Name]
+team_members_filter_description: List containing the criteria to filter team members
+[PROPOSED] team_members_typeahead: Filter by name
+team_members_typeahead_description: Filter by typing ahead the text
+
+team_members_table:
+  aria_label: Team members table
+  aria_label_description: Aria label of the table of team members
+  [PROPOSED] empty_state: Currently, there are no team members. Add team members.
+  empty_state_description: Message to show when there are no team members to show, and prompt the user to add members
```

Comment:
```suggestion
  empty_state_description: The message when there are no team members to show, and prompt the user to add members
```

Reply:
> The message when there are no team members to show, and prompt the user to add members

Shouldn't this be replaced by actual meaningful text?

Reply:
@thomasmaas 
The actual text is proposed in `empty_state`
`empty_state_description` is aimed at translators or localizers who in the future might need contextual information about the UI string.